security-analysis/security-analysis-api/src/test/resources/ActionFileTest.json

```diff
@@ -83,5 +83,31 @@
     "loadId" : "loadId1",
     "relativeValue" : true,
     "reactivePowerValue" : 5.0
+  }, {
+    "type" : "RATIO_TAP_CHANGER_TAP_POSITION",
+    "id" : "id14",
+    "transformerId" : "transformerId4",
+    "value" : 2,
+    "relativeValue" : false,
+    "side" : "THREE"
+  }, {
+    "type" : "RATIO_TAP_CHANGER_TAP_POSITION",
+    "id" : "id15",
+    "transformerId" : "transformerId5",
+    "value" : 1,
+    "relativeValue" : true
+  }, {
+    "type" : "RATIO_TAP_CHANGER_REGULATION",
+    "id" : "id16",
+    "transformerId" : "transformerId5",
+    "regulating" : true,
+    "side" : "THREE"
+  }, {
+    "type" : "PHASE_TAP_CHANGER_REGULATION",
+    "id" : "id17",
+    "transformerId" : "transformerId5",
+    "regulating" : false,
```

Comment:
In that case, the action asked to turn off the regulation. The regulation mode is not needed.